tox.ini

```diff
@@ -68,11 +68,35 @@ commands =
 [testenv:integration]
 description = Run integration tests
 deps =
-    juju
+    juju==2.9.11 # juju 3.3.0 has issues with retrieving action results
```

Comment:
did you meant `2.9.33` ? (the current stable in 2.x series)

Reply:
the highest 2.9 version for python-libjuju is `2.9.11` (i tried to change it to `2.9.33`, but got an error)